src/lib/get-participations.ts

```diff
@@ -0,0 +1,20 @@
+import type { Participations } from '@guardian/ab-core';
+import { isObject, isString, storage } from '@guardian/libs';
+
+const participationsKey = 'gu.ab._participations';
+
+const isParticipations = (
+	participations: unknown,
+): participations is Participations => {
+	if (!isObject(participations)) return false;
+	return Object.values(participations).some((participation) => {
+		if (!isObject(participation)) return false;
+		if (!isString(participation.variant)) return false;
+		return true;
+	});
+};
+
+export const getParticipationsFromLocalStorage = (): Participations => {
```

Comment:
Just to increase our understanding of this could we check if localstorage is set with all the tests the user participates in. Also if the localstorage participations includes server side and client side tests. Thank you!

Reply:
Yes definitely. Do you mean test on CODE? Or add unit tests here?

Reply:
Oh I just meant testing on CODE

Reply:
Confirmed localstorage participations is set with client-side tests _only_

Reply:
Great thanks for checking that!